perlmod/Fink/VirtPackage.pm

```diff
@@ -462,23 +465,28 @@ directories exist.
 					$java_inc_dir = $java_cmd_dir;
 					$java_inc_dir =~ s/bin/include/;
 				}
-				
-				
-				# chop the version down to major/minor without dots
-				$ver =~ s/$arch//g ; #strip architecture
-				$ver =~ s/[^\d]+//g ;
-				$ver =~ s/^(..).*$/$1/;
-				next if ($ver eq ""); # directories that aren't versioned;
-				$testver = $ver % 10; # Thanks for the directory renumber, oracle.
+				# chop the version down to major/minor without dots				
+				$ver =~ s/$arch//g ; # strip architecture
+			    $ver =~ s/,\s+//; # strip comma
```

Comment:
Indentation is wrong in multiple places here (tabs vs spaces). It's quite visible in the GitHub web view of the patch.